flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -466,6 +466,7 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
     }
 
     private func fetchNativeAppStart(result: @escaping FlutterResult) {
+        #if os(iOS) || os(tvOS)
```

Comment:
@vaind @brustolin This was available on macOS as well, this change will remove the feature on Flutter-macOS, I'd rather expose `appStartMeasurement` again for macOS on the Cocoa SDK, it's ok to be a quick fix, but I'd also open issues on the Cocoa SDK to fix this and later remove these checks on the Flutter SDK as well.

Reply:
Or @brustolin , both work on Cocoa.

Reply:
Oof, just saw this, sorry for the delay. Really there was no feature _functionality_ removed from Flutter-macOS, since the app start tracker did nothing on macOS: https://github.com/getsentry/sentry-cocoa/pull/3157/files#diff-d13c2b7f6a0357a11c275412ab996c4ce37cd3923403c9d287e36e8e27274470L49

I agree it's a pain that this kind of stuff is breaking stuff downstream. We need a way to test `sentry-cocoa` integration so we can avoid or be aware of future breakages.